### ISSUE TITLE
backport Rocky PRs to stable-2.9

### ISSUE THREAD

**Issue by rkm** [ansible/ansible#74565]
### Summary

Thank you for accepting support for Rocky in #74530 and #74545, and also backporting these to stable-2.10 and 11.

I understand that stable-2.9 is only accepting security fixes, however I see that similar patches for Alma Linux were recently backported to 2.9 (#73544). Ansible is only available in EPEL currently at version 2.9. Given that this will be the primary means for Rocky users to install Ansible, would you consider accepting backport PRs for Rocky to stable-2.9?

Thanks 🙂

### Issue Type

Bug Report

### Component Name

lib/ansible/module_utils/facts/system/distribution.py

### Ansible Version

```console
$ ansible --version
```

### Configuration

```console
$ ansible-config dump --only-changed
```

### OS / Environment

Rocky Linux 8.3

### Steps to Reproduce

N/A

### Expected Results

N/A

### Actual Results

```console
N/A
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by bcoca** [comment]
They were backported before the release of 2.11, which is what triggered the change of status for 2.9, you were just a bit too late.

**Comment by rkm** [comment]
Thanks, no problem. We'll attempt to get patches submitted downstream.

**Comment by samdoran** [comment]
What @bcoca said. Ansible Core 2.11 was released on 2021-04-26, putting Ansible 2.9 into 'security fix' only status. There are packages is EPEL for both [`ansible-core`](https://koji.fedoraproject.org/koji/packageinfo?packageID=33489) and [`ansible-base`](https://koji.fedoraproject.org/koji/packageinfo?packageID=32739), though those are only for Fedora. Contacting the package maintainer to ask about EL8 packages would probably help.
